src/main/java/redis/embedded/Cluster.java

```diff
@@ -0,0 +1,13 @@
+package redis.embedded;
+
+/**
+ * Created by dragan on 17.07.15.
+ */
+public interface Cluster extends Redis{
```

Comment:
We should think of naming it RedisCluster. However, what could be the name for current RedisCluster with sentinels? Also, Cluster should only define create() and take start/stop/... from Redis interface.

Reply:
Or just bump version of embedded-redis to `1.0`. Guys using 0.x versions will remain unaffected, and guys using new major version will have to do a little bit of work, which should be fine